docs/java-api/admin/indices/refresh.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[java-admin-indices-refresh]]
+==== Refresh
+
+The refresh API allows to explicitly refresh one or more index, making
+all operations performed since the last refresh available for search.
+The (near) real-time capabilities depend on the index engine used. For
+example, the internal one requires refresh to be called, but by default a
+refresh is scheduled periodically.
```

Comment:
I'd prefer to limit the number of cross-links so I simplified the wording instead of copy/paste the full doc.